src/Microsoft.ServiceFabric.Services.Remoting/V2/FabricTransport/Client/FabricTransportServiceRemotingClient.cs

```diff
@@ -105,6 +113,37 @@ public Task OpenAsync(CancellationToken cancellationToken)
         public async Task<IServiceRemotingResponseMessage> RequestResponseAsync(
             IServiceRemotingRequestMessage remotingRequestRequestMessage)
         {
+#if DotNetCoreClr

```

Comment:
Are we not supporting this in NetFx?

Reply:
Activity.cs is not supported by Net Framework